### DISCUSSION TITLE
Wallet Stacks

### DISCUSSION THREAD

**Discussion by tkuhrt** [openwallet-foundation/tac#123] [General]
In the March 6, 2024, TAC meeting, we discussed the idea of Wallet Stacks for OWF. This discussion thread is a space for us to discuss and provide feedback on this idea.

**Comment by jorgefl0** [comment]
Interesting to see mdoc proposed as a minimum requirement for the OWF stack. Is mdoc really an "open" standard? Would the average wallet user be able to obtain and understand the basic properties of an mdoc credential?

Also noteworthy to mention that the 2 wallets stacks currently under the purview of OWF are Credo and Bifold, whose currents strengths are based on AnonCreds and DIDComm.

**Comment by Sakurann** [comment]
where can I find the original list of wallet functions that Torsten proposed..?

**Reply by tkuhrt** [reply]
[Slide 17 of this deck](https://docs.google.com/presentation/d/1G-O26KtU2njSxBqwDn9A-RgE7ldRl5xT_2GnIknM0gI/edit#slide=id.g2bdd8129051_0_99) has the information you are looking for.
